src/compounds/RequestItem/RequestItem.jsx

```diff
@@ -35,11 +41,20 @@ const RequestItem = React.forwardRef(({ createdAt, description, href, img, title
 RequestItem.propTypes = {
   createdAt: PropTypes.string.isRequired,
   description: PropTypes.string.isRequired,
+  href: PropTypes.string,
   id: PropTypes.number,
+  index: PropTypes.number.isRequired,
   img: PropTypes.shape(Image.propTypes).isRequired,
   status: PropTypes.shape(Badge.propTypes).isRequired,
   title: PropTypes.string.isRequired,
   updatedAt: PropTypes.string.isRequired,
 }
 
+RequestItem.defaultProps = {
+  href: '',
+  id: null,
```

Comment:
we don't appear to be passing or using an `id` value in this component. the prop type described above must have been accidentally left over.

it should be removed instead, along with this default value.